cli/cobra.go

```diff
@@ -345,11 +407,20 @@ Examples:
 {{ .Example }}
 
 {{- end}}
+{{- if .HasParent}}
 {{- if .HasAvailableFlags}}
 
 Options:
 {{ wrappedFlagUsages . | trimRightSpace}}
 
+{{- end}}
+{{- end}}
+{{- if hasTopCommands .}}
+
+Common Commands:
```

Comment:
This could be `Commonly Used Commands` (or something along those lines) - open to suggestions